### PR TITLE
Support use of source tag in mustache

### PR DESCRIPTION
**Pull request by calebcordry**
the `src` attribute of a `source` tag is currently scrubbed by caja. This allows these attributes as long as the protocol is `https`

### CODE REVIEW

File: src/sanitizer.js [hunk 1]
```diff
@@ -198,7 +198,8 @@ const INVALID_INLINE_STYLE_REGEX =
  * @return {string}
  */
 export function sanitizeHtml(html) {
-  const tagPolicy = htmlSanitizer.makeTagPolicy();
+  const tagPolicy = htmlSanitizer.makeTagPolicy(parsed =>
+    parsed.scheme_ === 'https' ? parsed : null);
```
Comment (calebcordry) [inline src/sanitizer.js]: I had to change this a bit from using `isSecureUrl` since the callback gets a `URI` object, not an url

**Comment by jridgewell** [comment]
Adding @alabiaga as a required reviewer.

File: src/sanitizer.js (hunk 1 above)
Comment (alabiaga) [inline src/sanitizer.js]: do parsed.getScheme(), as scheme_ is a private field and it is best to access it via its API.

Comment (calebcordry) [reply src/sanitizer.js]: done.

**Comment by choumx** [comment]
@calebcordry Is there any context on why we require `source[src]` to be `https://`?

**Comment by calebcordry** [comment]
@choumx We just wanted it to be consistent with  the rules of `amp-video [src]`

**Pull request merged**